src/components/AddingFormInput/AddingFormInput.js

```diff
@@ -0,0 +1,15 @@
+export function AddingFormInput({ input, handleChange, listItem }) {
```

Comment:
Just 'Input' will be enough. 
```suggestion
export function Input ({ input, handleChange, listItem }) {
```